app/locales/base.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
```

Comment:
Can you please delete these files? Wev'e changed them so they'll be created only when pushing to `main`

Reply:
@yammesicka  - removed base.pot ; anything else to remove?